content/src/cloud/features/cloud-backup-download.md

```diff
@@ -0,0 +1,24 @@
+---
+zendesk:
+  article_id: 26294370569245
+  name: Downloading a backup from Home Assistant Cloud
+  position: 2
+  labels: cloud
+---
+
+## Prerequisites
+
+- You have administrator rights in Home Assistant
```

Comment:
You can also download it from the account page, which does not require Home Assistant to even be running.